cla_backend/apps/cla_provider/helpers.py

```diff
@@ -205,7 +205,11 @@ def _get_best_fit_provider(self, category):
         return self._get_random_provider(category)
 
     def get_suggested_provider(self, category):
-        if self.as_of not in settings.NON_ROTA_OPENING_HOURS:
+        if category.id == 8:
```

Comment:
Is there a way to make this check not be the id? Is there an appropriately computer-readable field relating to the category name on the model we could use instead?

Ideally, that field would also be used as the key in the `NON_ROTA_OPENING_HOURS` dict so then could be used directly and would scale nicer once we start getting more exceptions to the rule